src/lib.rs

```diff
@@ -608,6 +616,35 @@ impl<Types: IpfsTypes> Ipfs<Types> {
         rx.await?
     }
 
+    pub async fn bootstrap(&self) -> Result<(), Error> {
```

Comment:
```suggestion
    // Initiate a query for random key to discover peers.
    pub async fn bootstrap(&self) -> Result<(), Error> {
```

Is there a reason for this to be in the API or is it only while we are not doing this periodically ourselves?

Reply:
I don't know if we are doing it at all automatically; I can check

Reply:
doesn't seem that way, at least not in tests

Reply:
No we are not yet doing it automatically, but we probably should eventually.

What is the reason for exposing this as a method? Does kad queue these bootstraps or are they executed concurrently?

Reply:
> Does kad queue these bootstraps or are they executed concurrently?

good question; I don't know, but I'd expect it to be concurrent; a single call can result in a few of logs that the bootstrapping query has completed